app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -126,26 +126,16 @@ class TeamsAndRepositoriesConnector @Inject()(
   }
 
   def allTeams(implicit hc: HeaderCarrier): Future[Seq[Team]] =
-    http.GET[HttpResponse](teamsAndServicesBaseUrl + s"/api/teams").map {
-      toJson[Seq[Team]]
-    }
+    http.GET[Seq[Team]](teamsAndServicesBaseUrl + s"/api/teams")
 
   def allDigitalServices(implicit hc: HeaderCarrier): Future[Seq[String]] =
-    http.GET[HttpResponse](teamsAndServicesBaseUrl + s"/api/digital-services").map {
-      toJson[Seq[String]]
-    }
+    http.GET[Seq[String]](teamsAndServicesBaseUrl + s"/api/digital-services")
 
   def teamInfo(teamName: String)(implicit hc: HeaderCarrier): Future[Option[Team]] = {
     val url = teamsAndServicesBaseUrl + s"/api/teams_with_details/${URLEncoder.encode(teamName, "UTF-8")}"
 
     http
-      .GET[HttpResponse](url)
-      .map { response =>
-        response.status match {
-          case 404 => None
-          case 200 => Some(toJson[Team](response))
-        }
-      }
+      .GET[Option[Team]](url)
       .recover {
```

Comment:
Maybe we can get rid of this logging since http-verbs already does it https://github.com/hmrc/http-verbs/blob/master/src/main/scala/uk/gov/hmrc/http/logging/ConnectionTracing.scala#L29?